www/layouts/partials/sql-grammar/bnf/create-sink.bnf

```diff
@@ -1,9 +1,8 @@
 create_sink ::=
    'CREATE SINK' 'IF NOT EXISTS'? sink_name
    'FROM' item_name
-   'INTO' 'KAFKA BROKER' host 'TOPIC' topic
-   'FORMAT' 'AVRO USING'
-      ('CONFLUENT SCHEMA REGISTRY' url |
-      'SCHEMA'
-         ('FILE' path | inline_schema)
-      )
+   'INTO' (
```

Comment:
Unsure re: formatting here + the generated diagram is very wide not sure how to fix that

Reply:
<img width="1153" alt="Screen Shot 2020-04-06 at 12 17 20 PM" src="https://user-images.githubusercontent.com/4015012/78580623-a8782280-7800-11ea-9fa4-50f1dc48619e.png">

Reply:
@sploiselle maybe you know what to do here? I tried playing around with max width on the website to no avail

Reply:
This is sadly a limitation of the tool when one arm is very long and one arm is very short. The best solution for this is to do something hacky and break these up into multiple files. I'm glad to either do that for you or talk you through it.

Reply:
Lets talk through it - I'll message you separately

Reply:
Done